### ISSUE TITLE
Strange hover behavior for keyboard controls

### ISSUE THREAD

**Issue by mlangkabel** [CoatiSoftware/Sourcetrail#941]
When having the Sourcetrail windows not focussed, hovering the code view does not cause the graph view items to lose focus (the orange border) and the over way round. tested on Windows with Sourcetrail 2020.1.106 Beta.

**Comment by egraether** [comment]
Does this only happen when views are detached into separate windows?

**Comment by mlangkabel** [comment]
No, it happens when the dock widgets are docked in the main window but some other application has the current focus.

**Comment by egraether** [comment]
Then I'm not able to reproduce this on macOS
